src/pages/Thanks/ThanksPage.vue

```diff
@@ -185,20 +168,12 @@ export default {
 					checkoutId: transactionId,
 					visitorId: cookieStore.get('uiv') || null,
 				}
-			}).then(({ data }) => {
-				const isLoggedIn = data?.my?.userAccountId?.id !== null;
```

Comment:
I see that we still use the `thanksPageQuery` in `created` in this method, but are all of the fields from that query still needed? Might be worth doing a check on that query, since it looks like these consts were unused before, so we might be pulling data we no longer use.